tests/factories/filterFactory.js

```diff
@@ -0,0 +1,87 @@
+import faker from "faker";
+import { Factory, Field } from "./factory";
+
+export class FiltersFactory extends Factory {
+  getFields() {
+    return {
+      channels: Channels,
+      countries: [],
+      locales: [],
+    };
+  }
+
+  postGeneration() {
+    const { countries, locales } = this.options;
```

Comment:
i think these should be `generateCountriesCount`, `generateLocalesCount` and should have a default set.